cmd/cli/license/root.go

```diff
@@ -0,0 +1,19 @@
+package license
+
+import (
+	"github.com/bacalhau-project/bacalhau/cmd/util/hook"
+	"github.com/spf13/cobra"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "license",
+		Short: "Commands to inspect and validate local license information. To inspect an " +
+			"orchestrator license, please use 'bacalhau agent license inspect'.'",
```

Comment:
since this command is is only for local license inspection, then lets not introduce a flag and have the experience look like `bacalhau license inspect <path>`

Reply:
Change done: `bacalhau license inspect <path>`